### PR TITLE
Feature multi language

### PR DESCRIPTION
**Pull request by anhfactor**
add multi language

### CODE REVIEW

File: projects/marketing-site/public/lang/common.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+        },
+        {
+            "locale": "vi-VN",
+            "name": "Người sáng tạo nội dung",
+            "description":
+                "Sản phẩm cho phép người sáng tạo nội dung tuỳ chỉnh NFT và đăng bộ sưu tập của họ lên cửa hàng"
+        },
+        {
+            "locale": "vi-VN",
+            "name": "Bot Protection",
```
Comment (DistractionBoy) [inline projects/marketing-site/public/lang/common.json:129]: is there no real translation for "Bot Protection" in Vietnamese?

File: projects/marketing-site/public/lang/common.json [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+            "language": "Language",
+            "dark_mode": "Dark Mode",
+            "sign_out": "Sign out"
+        },
+        {
+            "locale": "vi-VN",
+            "vision": "Tầm nhìn",
+            "team": "Đội ngũ",
+            "funding": "Kinh phí",
+            "token": "Token",
```
Comment (DistractionBoy) [inline projects/marketing-site/public/lang/common.json:36]: is the word for Token the same?

Comment (anhfactor) [reply projects/marketing-site/public/lang/common.json:36]: Its the same, in vietnam we use word "token" can't translate

File: projects/marketing-site/public/lang/common.json (hunk 1 above)
Comment (anhfactor) [reply projects/marketing-site/public/lang/common.json:129]: Yes, i think so

**Review by DistractionBoy — APPROVED** [review]

**Pull request merged**